newton/sim/model.py

```diff
@@ -167,6 +167,37 @@ def __init__(self, device: Devicelike | None = None):
         self.muscle_activations = None
         """Muscle activations, shape [muscle_count], float."""
 
+        self.site_name = []
+        """List of site names."""
+        self.site_body = None
+        """Body index for each site, shape [site_count], int."""
+        self.site_xform = None
+        """Transform (position and orientation) for each site relative to its body, shape [site_count, 7], float."""
+
+        self.tendon_name = []
+        """List of tendon names."""
+        self.tendon_type = []
```

Comment:
The Warp arrays should be initialized with `None` like for the other Warp arrays in `Model`.